### PR TITLE
made product card

### PR DESCRIPTION
**Pull request by dmytro-kozlov**
- [DEMO LINK](https://dmytro-kozlov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://dmytro-kozlov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by PolinaSukhorukova** [comment]
Your links don't work

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-number">$2,199</p>
```
Comment (ihor-soloviov) [inline src/index.html:42]: "card__price-value" will be more appropriate

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+      </div>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >
+        byu
```
Comment (ihor-soloviov) [inline src/index.html:50]: change to "buy"

**Review by ihor-soloviov — COMMENTED** [review]
good job

**Pull request closed without merging**